shell/platform/darwin/macos/framework/Headers/FlutterEngine.h

```diff
@@ -21,7 +21,7 @@
  * Coordinates a single instance of execution of a Flutter engine.
  */
 FLUTTER_EXPORT
-@interface FlutterEngine : NSObject <FlutterTextureRegistry, FlutterPluginRegistry>
```

Comment:
Did you follow the breaking change process for this change? You've changed the public API of a public class.